src/snyk/extension.ts

```diff
@@ -273,6 +276,16 @@ class SnykExtension extends SnykLib implements IExtension {
     );
     this.ossVulnerabilityCountService.activate();
 
+    this.advisorScoreDisposable = new AdvisorScoreService(
+      vsCodeWindow,
+      vsCodeLanguages,
+      this.advisorService,
+      Logger,
+      vsCodeWorkspace,
+      this.advisorApiClient,
+    );
+    await this.advisorScoreDisposable.activate();
```

Comment:
Let's not `await` the function here. By removing it make sure you capture all possible errors within the activate function and all functions it calls.